.eslintrc.yml

```diff
@@ -0,0 +1,24 @@
+env:
+  browser: true
+  mocha: true
+  cypress/globals: true
+parserOptions:
+  ecmaVersion: 2015
```

Comment:
What about even newer versions? Although it would probably require Babel then?

Reply:
This setting doesn't seem to work anyway, VS code eslint extension complains about invalid ECMA version. Have to investigate. The JS code we use can be even run in IE9 I have tested ;)